src/taipy/config/checker/issue.py

```diff
@@ -31,3 +31,12 @@ class Issue:
     value: Any
     message: str
     tag: Optional[str]
+
+    def __str__(self) -> str:
+        message = self.message
+
+        if self.value:
+            current_value_str = f'"{self.value}"' if isinstance(self.value, str) else f"{self.value}"
+            message += f" Current value of property `{self.field}` is {current_value_str}."
+
```

Comment:
We cannot guarantee that self.value has is human readable. That is why we just show the message that must be clear enough.
I would remove these lines.
```suggestion
```

Reply:
If we don't use the Issue.field and Issue.value, then it is actually no use for those. I would recommend find a use-case for that 2 field, or get rid of those entirely.

Reply:
**About the 2 fields:** 
1 - It is a standart way of handling issues. 
2 - These 2 fields are mandatory for automatically processing the issues (a monitoring service, a no code software, ...). We don't have such a feature in taipy, but the issues are exposed to the users so they can use them, including these fields)
3 - In the future, Taipy Studio could benefit from these standardized attributes


**About the message:** 
I would remove the second part, but I believe it s not blocking. I approved the PR. If you believe we should keep it I am fine as well.

Reply:
I understand. As for now, I'll keep the second part since it is more informative for most cases.

If there is any weird error message in the future, this will be removed or adapted latter. I'll keep this in mind.